### DISCUSSION TITLE
Understanding "Last background job execution: never"

### DISCUSSION THREAD

**Discussion by Wordsmith9091** [nextcloud/recognize#977] [Q&A] [answered]
I'm running Recognize 4.3.2 on Nextcloud 27.0.2 (the apache Docker build), using it mostly via Memories 5.4.1.

I've got cron running by spinning up a second Nextcloud container in the same Docker Compose file as the rest of my setup, with the /cron.sh as an entrypoint, [like in this example](https://github.com/nextcloud/docker/blob/master/.examples/docker-compose/with-nginx-proxy/mariadb/apache/docker-compose.yml). Nextcloud seems to be working fine with cron, always showing last tasks executing within the last five minutes. I'm using Recognize in WASM mode, on an i5 chip with AVX instructions.

Recognize is clearly working -- it's finding and clustering new faces from time. I can easily see that within Memories. In the Recognize administration settings, I've got green boxes for "Face recognition is working," "object recognition is working" and "landmark recognition is working."

But below that are the yellow boxes, ie: "Face recognition: 0 Queued files, Last classification: 2 days ago, Scheduled background jobs: 0, **Last background job execution: never.**"

I'm unclear -- what is "last background job execution: never" telling me here? It's clearly run past classifications. The very first was one I manually triggered, but there have been more since then. So if those aren't background job executions ... what are they?

**Comment by marcelklehr** [comment, marked as answer]
Mmh. Interesting. Once all new files have been processed, the background job is removed from Nextcloud's job list, which also removes the information when it last ran, giving the appearance that it ran never, even though it did run. Thanks for reporting this, I'll change this
